### PR TITLE
Remove Bootstrap css from App.vue page

### PR DESCRIPTION
**Pull request by cp-shruti-s**
Remove Bootstrap css from App.vue page

### CODE REVIEW

File: vue-frontend/src/App.vue [hunk 1]
```diff
@@ -33,181 +33,96 @@ export default {
 };
 </script>
 
-<style lang="scss">
+<style lang="postcss">
 @import "bootstrap/dist/css/bootstrap.min.css";
```
Comment (cp-sumi-k) [inline vue-frontend/src/App.vue:37]: Do we need this now?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/App.vue [hunk 2]
```diff
@@ -216,280 +131,75 @@ a {
...
 
 .swiper-pagination-bullet {
-  width: 15px !important;
+  /*width: 15px !important;
   height: 15px !important;
   border-radius: 3px !important;
   background: transparent !important;
-  border: 1px solid !important;
+  border: 1px solid !important;*/
+
```
Comment (cp-priyanka-g) [inline vue-frontend/src/App.vue:143]: Remove comments

File: vue-frontend/src/App.vue [hunk 3]
```diff
@@ -216,280 +131,75 @@ a {
...
 .swiper-pagination-bullet-active {
-  border: none !important;
+  /*border: none !important;
   background: linear-gradient(
     45deg,
     rgb(246, 146, 89),
     rgb(241, 105, 117)
-  ) !important;
+  ) !important;*/
+
```
Comment (cp-priyanka-g) [inline vue-frontend/src/App.vue:154]: Remove comments.

**Review by cp-priyanka-g — APPROVED** [review]

File: vue-frontend/src/App.vue (hunk 1 above)
Comment (cp-sumi-k) [reply vue-frontend/src/App.vue]: Shouldn't we need to uninstall bootstrap?

File: vue-frontend/src/components/home-new/ContributeSection.vue [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+        here to
+      </div>
       <div class="v2-title-3-text">
-        <span class="v2-header-2-text pe-2">contribute</span>
+        <span class="v2-header-2-text tw-font-futura-bold pe-2"
+          >contribute</span
+        >
         and that starts with the site itself,
       </div>
       <div class="v2-title-3-text mt-1">
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/home-new/ContributeSection.vue:14]: Add tw prefix in mt-1

File: vue-frontend/src/components/jobs/VirtuesView.vue [hunk 5]
```diff
@@ -277,7 +277,7 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped>
+<style lang="postcss" scoped>
 .descriptions {
   transform: translateX(-50%);
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/VirtuesView.vue:282]: Shouldn't we need to apply tailwind classes here?

File: vue-frontend/src/pages/portfolio/[id].vue [hunk 6]
```diff
@@ -120,8 +123,8 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped>
-@include media-breakpoint-up(lg) {
+<style lang="postcss" scoped>
+@screen lg {
   .parallax {
     height: 100vh;
```
Comment (cp-sumi-k) [inline vue-frontend/src/pages/portfolio/[id].vue:129]: Shouldn't we need to apply tailwind classes here?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/pages/portfolio/[id].vue (hunk 6 above)
Comment (cp-shruti-s) [reply vue-frontend/src/pages/portfolio/[id].vue:129]: Perspective property is not in tailwind css

Comment (cp-sumi-k) [reply vue-frontend/src/pages/portfolio/[id].vue:129]: https://stackoverflow.com/a/49993773
https://tailwindcss.com/docs/translate
Refer these links

Comment (cp-shruti-s) [reply vue-frontend/src/pages/portfolio/[id].vue:129]: but that perspective css property is not tailwind css

File: vue-frontend/src/components/home-new/BlogSection.vue [hunk 7]
```diff
@@ -21,8 +21,9 @@
             </div>
             <img
               :src="backgroundImage"
-              class="tw-hidden md:tw-block tw-pt-12"
+              class="tw-hidden md:tw-block tw-pt-12 tw-h-full tw-w-[260px]"
               loading="lazy"
+              alt="dots-blog-bckground"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/BlogSection.vue:26]: Rename it to `blog-background`

File: vue-frontend/src/components/home-new/WithCanopas.vue [hunk 8]
```diff
@@ -90,30 +92,35 @@ export default {
...
         {
           image: designing,
+          alt: "designing-animation",
           title: "Design",
           description:
             "Working passionately with you, our designers create aesthetically pleasing  and user friendly digital experiences.",
         },
         {
           image: development,
+          alt: "development-animation",
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/WithCanopas.vue:109]: Remove `-animation ` suffix from all

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**